src/ucs/stats/stats.c

```diff
@@ -98,6 +98,11 @@ static ucs_stats_class_t ucs_stats_root_node_class = {
     }
 };
 
+/* Statistics id to name */
+static ucs_stats_class_id_to_name_s
+        g_stats_class_id_to_name[MAX_NUM_OF_STATS_CLASSES];
+static size_t g_stats_class_id_to_name_size;
+static size_t g_stats_class_id_to_name_last_offset;
```

Comment:
I think this could easily fit inside the stats context (which is basically a global too :) )

Reply:
Fixed.